.devcontainer/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# syntax=docker/dockerfile:1.5
+
+ARG CUDA=11.8
+ARG LLVM=15
+ARG RAPIDS=23.04
+ARG DISTRO=ubuntu22.04
+ARG REPO=rapidsai/devcontainers
+
+ARG PYTHON_PACKAGE_MANAGER=conda
+
+FROM ${REPO}:${RAPIDS}-cpp-llvm${LLVM}-cuda${CUDA}-${DISTRO} as pip-base
+
+FROM ${REPO}:${RAPIDS}-cpp-llvm${LLVM}-cuda${CUDA}-mambaforge-${DISTRO} as conda-base
+
+FROM ${PYTHON_PACKAGE_MANAGER}-base
+
+ARG PYTHON_PACKAGE_MANAGER
+ENV PYTHON_PACKAGE_MANAGER="${PYTHON_PACKAGE_MANAGER}"
+
+USER coder
+
+COPY --chown=coder:coder opt/cuspatial /opt/cuspatial
+
+RUN /bin/bash -c 'mkdir -m 0755 -p ~/{.aws,.cache,.conda,.config,.local}'
+
+WORKDIR /home/coder/
+
+ENV PYTHONDONTWRITEBYTECODE="1"
+
+ENV SCCACHE_REGION="us-east-2"
+ENV SCCACHE_BUCKET="rapids-sccache-devs"
+ENV VAULT_HOST="https://vault.ops.k8s.rapids.ai"
```

Comment:
Is it possible to set these values somewhere higher up in the images?

It'd be nice to have them centralized somewhere instead of in the individual repositories in case they ever change.

Reply:
Yes it's possible, but I've avoided that since the base images aren't specific to RAPIDS or even NVIDIA, so I don't want to bake in our values.

Reply:
I think we could have vault return the sccache bucket and region as part of the `GET /v1/aws/creds/devs` response, and if so then we would only need the `VAULT_HOST` envvar here.